vnext/Microsoft.ReactNative.Cxx/JSValueTreeWriter.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#ifndef MICROSOFT_REACTNATIVE_JSVALUETREEWRITER
```

Comment:
>MICROSOFT_REACTNATIVE_JSVALUETREEWRITER [](start = 8, length = 39)

Why do we need a header guard if we have `#pragma once`?

Reply:
Pragma once only protects from multiple use of code if it came from the same file. If the same file came from different source locations due to symlinking or because it was copied, then it will not protect. It is generally safer to provide the guard macros in all shared header files.
The gurad does not replace "pragma once" because the pragma enables compiler not even open the file second time.

---
In reply to: [356918446](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356918446) [](ancestors = 356918446)